Project1/Task10-homework/Collections1.cs

```diff
@@ -0,0 +1,243 @@
+            //4)Modify your code to prompt the user to enter the length of a search term.
+            Console.WriteLine("List --3--");
+            Console.WriteLine("Write the length of word:");
+            int length = int.Parse(Console.ReadLine());
+            FindTheWord(myListS, length);
+
+            //Create a LinkedList and two items, insert a second item after each occurrence of the first item in the list.So, if the list is [2,4,3,2,8,2,5,1,2] and the elements are 2 and 10, the result is [2,10,4,3,2,10,8,2,10,5,1,2,10] 
+            Console.WriteLine("LinkedList --1--");
+            var initList = new List<int> { 2, 4, 3, 2, 8, 2, 5, 1, 2 };
+            LinkedList<int> linkedList = new LinkedList<int>(initList);
+            int a = 2; int b = 10;
+            int i = 0;
+            int count = 0;
+            LinkedList<int> list1 = new LinkedList<int>();
+            while (i < linkedList.Count)
+            {
+                if (linkedList.ElementAt(i) == a)
+                {
+                    var newNode = linkedList.ElementAt(i);
+                    initList.Insert(i + 1, b);
+                }
+                i++;
+            }
+            int k = initList.Count - 1;
+            if (initList.ElementAt(k) == a)
+            {
+                initList.Add(10);
+            }
+            foreach (var stu in initList)
+            {
+                Console.WriteLine(stu);
+            }
+            //Merge two related lists of numbers by including in the final list only those numbers that occur in both lists. So, if the lists are[1, 3, 4, 7, 12] and[1, 5, 7, 9], the result will be[1, 7]. *
+            Console.WriteLine("List --2--");
+            var finalList = new List<int>();
+            var firstList = new List<int> { 1, 3, 4, 7, 12 };
+            var secondList = new List<int> { 1, 7 };
+            for (int j = 0; j < firstList.Count; j++)
+            {
+                for (int l = 0; l < secondList.Count; l++)
+                {
+                    if (firstList[j] == secondList[l])
+                    {
+                        finalList.Add(firstList[j]);
+                    }
+                }
+            }
+            foreach (var item in finalList)
+            {
+                Console.WriteLine(item);
+            }
+
+            //Create a Queue via inputting queue elements into console.Create GetMaxValue() method, which should return the maximum value stored in the queue and not remove this value from the queue. Create DeleteMaxValue() method which find max value in the queue and removes it.Call GetMaxValue() method => confirm it returns max value and not deletes it from the queue.Call DeleteMaxValue () => confirm it deletes max value from the queue.Call GetMaxValue() again and confirm it returns actual max value.
+            Console.WriteLine("queue:");
+            Queue<int> queue = new Queue<int>();
+            Console.WriteLine("add element ");
```

Comment:
88 and 89 lines duplicate by 3 times, so I think it's better to move them out and create private method